src/cpp/main.cpp

```diff
@@ -90,14 +93,14 @@ struct RunConfig {
   bool useCuda = false;
 };
 
-void parseArgs(int argc, char *argv[], RunConfig &runConfig);
-void rawOutputProc(vector<int16_t> &sharedAudioBuffer, mutex &mutAudio,
-                   condition_variable &cvAudio, bool &audioReady,
-                   bool &audioFinished);
+void parseArgs(int argc, char* argv[], RunConfig& runConfig);
```

Comment:
Style changes for unrelated code should not be in this PR. If a formatter should be used, we should put that in a different PR.

Reply:
Fair enough, I've created another containing just the formatting changes: https://github.com/rhasspy/piper/pull/744